### PR TITLE
Changed rectangular buttons to circular

### PR DESCRIPTION
**Pull request by S-ulphuric**
Changed rectangular buttons to circular

### CODE REVIEW

File: frontend/src/css/modules/base.scss [hunk 1]
```diff
@@ -415,6 +415,7 @@ nav {
 .btn {
     text-transform: none;
     font-family: 'Roboto';
+    border-radius: 20px;
```
Comment (aka-jain) [inline frontend/src/css/modules/base.scss]: @S-ulphuric There is already a class '.grad-btn' please add that and remove this style form here. You have to find those buttons to make these required changes.

Comment (S-ulphuric) [reply frontend/src/css/modules/base.scss]: Ok. Thanks

**Comment by S-ulphuric** [comment]
Please review changes

**Comment by S-ulphuric** [comment]
Thanks!
Please approve the task.

**Pull request merged**